wandb/sdk/launch/sweeps/scheduler_sweep.py

```diff
@@ -94,6 +83,9 @@ def _heartbeat(self, worker_id: int) -> None:
                 # The command "type" can be one of "run", "resume", "stop", "exit"
                 _type = command.get("type", None)
                 if _type in ["exit", "stop"]:
+                    if command.get("run_cap", 0) > 0:
+                        # over runcap: dont stop scheduler yet
```

Comment:
It almost seems like we want to put the scheduler into a state where it just flushes the existing runs and lets them run to completion here? Perhaps we can introduce a specific function for that? Am I understanding the intent of this PR right?